packages/react/changelog.yaml

```diff
@@ -0,0 +1,11 @@
+unreleased:
+  type: minor
+
+  changed:
+    - summary: Initialize @cerbos/react, A collection of React hooks for interacting with Cerbos policy decision points.
+      pull: 876
+
+references:
+  nodejs-eol: https://github.com/nodejs/release#end-of-life-releases
+
```

Comment:
Let's just drop this for now, we won't be dropping support for any Node.js versions until 18 is EOL at the end of April next year.

```suggestion
```